factory/crypto/cryptoComponents.go

```diff
@@ -294,10 +294,10 @@ func (ccf *cryptoComponentsFactory) createCryptoParams(
 			return nil, fmt.Errorf("invalid node configuration: import-db mode and allValidatorsKeys.pem file provided")
 		}
 
-		return ccf.generateCryptoParams(keygen, "in import mode", handledPrivateKeys)
+		return ccf.generateCryptoParams(keygen, "in import mode", make([][]byte, 0))
 	}
 	if ccf.noKeyProvided {
-		return ccf.generateCryptoParams(keygen, "with no-key flag enabled", handledPrivateKeys)
+		return ccf.generateCryptoParams(keygen, "with no-key flag enabled", make([][]byte, 0))
```

Comment:
When I first wrote the code I thought that I can provide a `-no-key` flag but will still use the allValidatorsKeys file, if existed. 
No, I think it is better to not use any of the provided keys if the flag was set to `-no-key`. So yes, it will remain like this :+1: